src/scripts.js

```diff
@@ -5,7 +5,46 @@
 import './css/styles.css';
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
-import './images/turing-logo.png'
+// import './images/turing-logo.png'
 
+// Fetch Requests
 
-console.log('This is the JavaScript entry file - your code begins here.');
+// Local Data
+import TripsClass from './trips-class';
+import { getData } from './apiCalls';
+import { postData } from './apiCalls';
+
+
+function getRandomInt() {
+  return Math.floor(Math.random() * 49 + 1);
+};
+const randomNum = getRandomInt();
+
+window.addEventListener('load', () => {
+  getData('travelers').then((data) => {
+    const welcomeUser = document.querySelector('.user-welcome')
```

Comment:
Interesting to see you assigning the query selectors within the function. I wouldn't have thought to structure it this way!